USAGE.md

```diff
@@ -140,3 +140,15 @@ funders: GBMF4552  # optional
 ## Manubot feedback
 
 If you experience any issues with the Manubot or would like to contribute to its source code, please visit [`greenelab/manubot`](https://github.com/greenelab/manubot) or [`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock).
+
+## Examples
+
+For additional examples, check out existing manuscripts that use the Manubot:
+
++ The Sci-Hub Coverage Study ([source](https://github.com/greenelab/scihub-manuscript), [manuscript](https://greenelab.github.io/scihub-manuscript/))
++ A Report for the Vagelos Scholars Program by Michael Zietz ([source](https://github.com/zietzm/Vagelos2017), [manuscript](https://zietzm.github.io/Vagelos2017/))
++ The Deep Review ([source](https://github.com/greenelab/deep-review), [manuscript](https://greenelab.github.io/deep-review/))
++ The Meta Review ([source](https://github.com/greenelab/meta-review), [manuscript](https://greenelab.github.io/meta-review/))
```

Comment:
I think it is fine to link the meta review here because it's already public.  We can update the name once we finalize it.  We should confirm with @cgreene as well that this is okay to share.

Reply:
Yes - fine with me :+1: